### PR TITLE
* Begone using statements - pt. 1

### PR DESCRIPTION
**Pull request by Wagnerp**
* Some C# 10 stuff (#117)

### CODE REVIEW

File: Source/Krypton Components/Krypton.Docking/GlobalDeclarations.cs [hunk 1]
```diff
@@ -1,4 +1,18 @@
...
+﻿#region BSD License
+/*
+ * 
+ * Original BSD 3-Clause License (https://github.com/ComponentFactory/Krypton/blob/master/LICENSE)
+ *  © Component Factory Pty Ltd, 2006 - 2016, All rights reserved.
+ * 
+ *  New BSD 3-Clause License (https://github.com/Krypton-Suite/Standard-Toolkit/blob/master/LICENSE)
+ *  Modifications by Peter Wagner(aka Wagnerp) & Simon Coghlan(aka Smurf-IV), et al. 2017 - 2021. All rights reserved. 
+ *  
+ *  Modified: Monday 12th April, 2021 @ 18:00 GMT
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Docking/GlobalDeclarations.cs:10]: What is this modified date for ?

File: Source/Krypton Components/Krypton.Toolkit/General/PlatformInvoke.cs [hunk 2]
```diff
@@ -23,6 +23,15 @@
 #pragma warning disable 649
 
 
+using System;
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/General/PlatformInvoke.cs:26]: Why are these in here, if there is supposed to be a removal taking place ?

File: Source/Krypton Components/Krypton.Ribbon/GlobalDeclarations.cs [hunk 3]
```diff
@@ -20,6 +20,7 @@
 global using System.ComponentModel.Design;
 global using System.Diagnostics.CodeAnalysis;
 global using System.Drawing;
+global using System.Drawing.Imaging;
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Ribbon/GlobalDeclarations.cs:23]: Alphabetical please ?

File: Source/Krypton Components/Krypton.Toolkit/General/PlatformInvoke.cs (hunk 2 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/General/PlatformInvoke.cs:26]: The problem here is that this particular class gets confused on what resources to access. Unsure why, as they're not needed.

File: Source/Krypton Components/Krypton.Ribbon/GlobalDeclarations.cs (hunk 3 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Ribbon/GlobalDeclarations.cs:23]: Will do.

File: Source/Krypton Components/Krypton.Docking/GlobalDeclarations.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Docking/GlobalDeclarations.cs:10]: Will remove, it was part of the generic header template.

File: README.md [hunk 4]
```diff
@@ -112,6 +112,7 @@ Please follow [this](https://github.com/Krypton-Suite/Standard-Toolkit/blob/alph
 =======
 
 ## 2021-08-01 Build 2108 - August 2021 (Canary)
+* Removed `KryptonStatusStrip` & `KryptonToolStrip` - to be implemented in the `Extended Toolkit`
```
Comment (Smurf-IV) [inline README.md:115]: Not sure that should be the case, as `ToolStrip` and `Status strip`'s are part of the standard `Winform ToolBox`.

**Pull request merged**